supervisord.conf

```diff
@@ -0,0 +1,14 @@
+[supervisord]
```

Comment:
What is this file?

Reply:
This is needed for the docker image, as we are running multiple processes.
It is added when building the image, so it can't be in a `.docker-ignore` directory.
We should probably find a better place for it,  maybe `_assets` or something? 

`docker-build` is now in `.dockerignore` so it can't be there ( We can also change the dockerignore or shuffle things around).